app/serializers/group_serializer.rb

```diff
@@ -36,11 +36,11 @@ def include_logo_url_medium?
   end
 
   def cover_url_desktop
-    object.cover_photo.url(:desktop)
-  end
-
-  def include_cover_url_desktop?
-    object.cover_photo.present?
+    if object.cover_photo.present? || object.default_group_cover.present?
+      object.cover_photo.url(:desktop)
+    else
+      'img/default-cover-photo.png'
```

Comment:
This is because the default for angular and the default for beta are different at the moment (`img/default-cover-photo.png` vs `default-cover-photo.png`)

Is this a great place to put this? No. Are we going to make the move at some point and apply default cover photos to all existing groups which don't have cover photos? I think so, so can go away when that happens.